src/conf.rs

```diff
@@ -72,8 +72,9 @@ impl Default for AccelDataRate {
 impl AccelDataRate {
     pub(crate) fn accel_config_bits(&self) -> u8 {
         match self {
-            AccelDataRate::FChoice0 => 0b00010000,
-            AccelDataRate::DlpfConf(dlpf) => 0b00000000 | (*dlpf as u8),
+            AccelDataRate::FChoice0 => 0b0000_1000,
+            // 0x40 for 1024 bit fifo
```

Comment:
Is it possible to split out formatting changes? Also, looks like rustfmt.toml committed to repo doesn't produce this change.

Reply:
Ok for the formatting. For the line 75, I have shifted the value to 1 to the left according to the documentation of the register map page 15 which says that `FChoice0` is bit 3. If you also want me to keep this change for latter tell me.